msg/ko_KR.utf8/cubrid.msg

```diff
@@ -1431,8 +1431,9 @@ $ LOADDB
 1350 잘못된 Synonym 값입니다.
```

Comment:
ko_KR.utf8 should be displayed normally, but characters are broken after 1350.

Reply:
The cause is unknown, but it seems to be a problem with the screen display.
If you receive the actual file and look at it, it's normal.